src/pkg/bundle/create.go

```diff
@@ -107,3 +114,60 @@ func (b *Bundle) confirmBundleCreation() (confirm bool) {
 	}
 	return true
 }
+
+// processValuesFiles reads values from valuesFiles and updates the bundle with the override values
+func (b *Bundle) processValuesFiles() error {
+	// Populate values from valuesFiles if provided
+	for i, pkg := range b.bundle.Packages {
+		for j, overrides := range pkg.Overrides {
+			for k, bundleChartOverrides := range overrides {
+				for _, valuesFile := range bundleChartOverrides.ValuesFiles {
+					// Check relative vs absolute path
+					fileName := filepath.Join(b.cfg.CreateOpts.SourceDirectory, valuesFile)
+					if filepath.IsAbs(valuesFile) {
```

Comment:
Noting that we do this in other areas of the codebase (handling absolute vs relative file paths), will make a follow-up issue to refactor into a single fn